src/types/dsnpTypes.ts

```diff
@@ -0,0 +1,103 @@
+import { EthereumAddress } from "@liberty30/lib-privacy-js";
```

Comment:
We will want to just pull this in instead. Also with TS 4.1 we can do template literal types! (https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html)

```typescript
type EthereumAddress = `0x${string}`;
```